src/arrayMethodJoin.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let resultStr = '';
+
+    if (separator === undefined) {
+      separator = ',';
+    }
+
+    separator = separator === null ? 'null' : separator.toString();
```

Comment:
create a variable with result of `String(separator)`

Reply:
then use it

Reply:
Done